python/sglang/srt/openai_api/adapter.py

```diff
@@ -1031,11 +1050,51 @@ def v1_chat_generate_response(request, ret, to_file=False, cache_report=False):
 
         finish_reason = ret_item["meta_info"]["finish_reason"]
 
+        tool_calls = None
+        text = ret_item["text"]
+
+        if isinstance(request, list):
+            tool_choice = request[idx].tool_choice
+            tools = request[idx].tools
+        else:
+            tool_choice = request.tool_choice
+            tools = request.tools
+
+        if tool_choice != "none" and (
+            "<|plugin|>" in text
```

Comment:
1. Make these constants a list of constant strings
2. Put this list above `parse_tool_response` in `python/sglang/srt/utils.py`
3. Import this list here.

After this, people only need to update one file to support a new model.